conf/edgemanage.yaml

```diff
@@ -103,6 +103,9 @@ live_list: /var/tmp/edges.{dnet}.live
 # names to IP addresses.
 canary_files: /etc/edgemanage/canaries/{dnet}
 
+# prometheus_client log file path (with trailing /)
+prometheus_logs: /var/log/prom/
```

Comment:
Hi @jeremy5189, just a minor suggestion; I think it would be a good idea to use `/var/log/prometheus` instead of just `prom` for the default metrics directory. This is not a blocker for merging the PR.